Library/Formula/libgig.rb

```diff
@@ -0,0 +1,35 @@
+class Libgig < Formula
+  desc "library for Gigasampler and DLS (Downloadable Sounds) Level 1/2 files"
+  homepage "http://www.linuxsampler.org/libgig/"
+  url "http://download.linuxsampler.org/packages/libgig-3.3.0.tar.bz2"
+  sha256 "d22a3c7ba13d920c1d4b6c218107ad105622ae9d1236ffbce007b98547774425"
+
+  depends_on "pkg-config" => :build
+  depends_on "libsndfile"
+
+  def install
+    ENV.deparallelize
```

Comment:
I've added comments for that. Yes, the latest version would be probably be better, but for me this is currently sufficient. Macports also hasn't 4.0.0 running and I want avoid being the guinea pig here.